src/main/java/no/uib/inf112/core/player/Robot.java

```diff
@@ -14,6 +14,7 @@
 
 public abstract class Robot extends AbstractRequirementTile implements Entity {
 
+    private boolean stopMoving = false;
```

Comment:
Is there a test for the `stopMoving` logic?

Reply:
No there is not a test directed towards this, but it exists to maintain expected functionality. E.g player walking on a whole should stop movement. Its difficult to test directly as it would require multithreading. Breaking this functionality would break expected(and tested) behavior of stepping on tiles and doing no more movement, and no more movement after walking of the map.

Reply:
Does it really need multi threading? Why not kill make the robot move over the edge then check if it is at the respawn location.

Reply:
Something like this
```java
@Test
public void stopMovingWhenKilled() {
    player.setBackup(1, 1);
    player.teleport(0, 0);
    player.setDirection(Direction.SOUTH);
    player.move(Movement.MOVE_2);
    assertEquals(1, player.getX());
    assertEquals(1, player.getY());
}
```

Reply:
https://github.com/inf112-v19/Blodige-roboter/blob/002a141f625432f87900e9810e3a4218aa777ccd/src/test/java/no/uib/inf112/core/map/tile/tiles/DeathTileTest.java#L46-L53, isn't this good enough?

Reply:
No in that test the player will get killed twice

Reply:
https://github.com/inf112-v19/Blodige-roboter/blob/b25279ece6c2385913d8aafdfb257f8b3053efdb/src/test/java/no/uib/inf112/core/map/tile/tiles/DeathTileTest.java#L46-L55

Reply:
Now it is not clear why it failed, maybe add a comment